sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/ScalarFunctionImpl.java

```diff
@@ -91,14 +93,15 @@ private ScalarFunctionImpl(Method method, CallImplementor implementor) {
    *
    * @param clazz class that is used to implement the function
    * @param methodName Method name (typically "eval")
+   * @param funGroup Optional function group identifier to differentiate implementations.
```

Comment:
Not sure this clarifies what a "function group" is. What is it?

Reply:
It's an arbitrary ZetaSQL field: https://github.com/google/zetasql/blob/1e03a558e2c26ad8f6a47fcb388ac4265d92f0dd/java/com/google/zetasql/Function.java#L61

This probably won't be needed if we use a custom interface instead of Method, since then we will need a new and separate implementation.